### ISSUE TITLE
Windows receives but don't show output

### ISSUE THREAD

**Issue by fsuarezo** [enclave-networks/multicast-test#5]
We've compiled the source to run on Linux, and we've tried sending and receiving successfully.
But when we try to receive from Windows, the program is silent at: "Bound udp listener on 10.2.192.59. Joined multicast group 239.0.1.2. Waiting to receive data...".
Using Wireshark we can see that packets are coming, so we don't really know if multicast-test is just "not showing what it's receiving" or simply "nor receiving at all".
![image](https://user-images.githubusercontent.com/116004421/211075337-9fd5d730-db3b-446e-9eaf-05b3cbdb5640.png)

**Comment by enclave-marc-barry** [comment]
Most likely not receiving at all. Is that running in WSL?

**Comment by fsuarezo** [comment]
No, it's a Windows Server 2019 VM. The interesting part is that packets are being received at the interface because Wireshark shows them (the hex dump showed in the capture is the text that the producer is creating).

**Comment by enclave-marc-barry** [comment]
Did you ever get to the bottom of this @fsuarezo?
